docs/examples/demoscenario.md

```diff
@@ -0,0 +1,115 @@
+
+4. Click **Add once** button to provide ServiceInstance for **github-connector** ServiceClass. Create it with Default Plan and provided name.
+	![Service instance](./pictures/demoscenario-09-service-instance.png)
+
+5. In the Instance Catalog under **Services** tab you can find the newly created instance and its status.
+	![Service instance](./pictures/demoscenario-10-service-instance-status.png)
+
+6. In the **Lambdas** catalog in namespace click **+ Add Lambda** to start creating lambda function.
+	![Create lambda](./pictures/demoscenario-11-create-lambda.png)
+
+#### Creation of Lambda function
+
+After clicking **Add Lambda** you will be able to select triggering options and write body of your function.
+
+1. Choose function name and click **Select Funtion Trigger** to choose the trigger method.
+	![Select function trigger](./pictures/demoscenario-12-select-function-trigger.png)
+2. From the drop-down menu that appears choose the trigger option you desire. For the purpose of this example **Event Trigger** is chosen.
+	![Choose function trigger](./pictures/demoscenario-13-choose-function-trigger.png)
+3. Select events for which this specific lambda you desire to be able to react.
+	![Select events](./pictures/demoscenario-14-choose-events.png)
+4. Create service binding and choose your service. Click **Create Service Binding** to apply binding.
+   	![Write lambda](./pictures/demoscenario-15-create-service-binding.png)
+
+5. Write body of your function and click **Create**.
+	![Write lambda](./pictures/demoscenario-16-write-lambda.png)
+
+Everything should be working now.
+
+### Option 2. Shell script
+
+1. Go to the `/chart/demoscenario` directory, where you can find `demoscenario.sh` script.
+
+2. Run the script and supply the name, which has to be the same as the one you used as release name while installing GitHub Connector with helm, and the namespace in which GitHub Connector application is running.
+
+   ```shell
+   sh demoscenario.sh {NAME} {NAMESPACE}
+   ```
+
+   After you trigger the script you get the following output that shows the created resources:
+
+   ```
+   applicationmapping.applicationconnector.kyma-project.io/gh-connector-example-app created
+   serviceinstance.servicecatalog.k8s.io/gh-connector-example created
+   function.kubeless.io/gh-connector-example-lambda created
+   subscription.eventing.kyma-project.io/gh-connector-example-lambda-issuesevent-opened-v1 created
+   Subscribed! Happy GitHub Connecting!
+   ```
+
+   Now your GitHub Connector is configured to react to new issues opened on GitHub repository you have connected to during Connector installation.
+
+   To test if connetion works, create a new issue on connected repository to trigger the event in lambda. Then, run this command to find lambda's name (it should consist of name you have provided earlier and *-lambda* suffix):
+
+   `kubectl get pods -n {NAMESPACE}`
+
+   Copy the output and paste it into this command to check lamda's logs:
```

Comment:
The output of `kubectl get pods -n {NAMESPACE}` is the list of all Pods in the Namespace. Are you sure I should copy this entire list over to the next command?

Reply:
Good point, fixed.